native/src/main/java/io/ballerina/stdlib/persist/ErrorGenerator.java

```diff
@@ -0,0 +1,33 @@
+package io.ballerina.stdlib.persist;
```

Comment:
Shall we add the license header?

Reply:
Fixed in [fe83294](https://github.com/ballerina-platform/module-ballerina-persist/pull/245/commits/fe83294a1026025e198b5b9cb90960ac1e36da4d)